src/zope/traversing/api.py

```diff
@@ -26,22 +29,8 @@
 
 
 def joinPath(path, *args):
-    """Join the given relative paths to the given path.
-
-    Returns a unicode path.
-
-    The path should be well-formed, and not end in a '/' unless it is
-    the root path. It can be either a string (ascii only) or unicode.
-    The positional arguments are relative paths to be added to the
-    path as new path segments.  The path may be absolute or relative.
-
-    A segment may not start with a '/' because that would be confused
-    with an absolute path. A segment may not end with a '/' because we
-    do not allow '/' at the end of relative paths.  A segment may
-    consist of . or .. to mean "the same place", or "the parent path"
-    respectively. A '.' should be removed and a '..' should cause the
-    segment to the left to be removed.  joinPath('/', '..') should
-    raise an exception.
+    """
+    Join the given relative paths to the given path.
```

Comment:
~~Did you mean to delete all this text?~~

This seemed peculiar before I saw the interface `__doc__` copying at the end of the module.